src/pydvl/utils/dataset.py

```diff
@@ -172,17 +202,28 @@ def from_sklearn(
         data: Bunch,
         train_size: float = 0.8,
         random_state: Optional[int] = None,
+        stratify: bool = False,
```

Comment:
This might be confusing for people expecting the same interface as sklearn [`train_test_split`](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html)'s interface where the `stratify` argument is expected to be an array of the same size as the labels and not a boolean variable.